shared/common-adapters/standard-screen.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../styles/style-guide'
+import type {Props} from './standard-screen'
+
+const defaults = {
+  showClose: true,
+}
+
+const StandardScreen = (props: Props) => {
+  const {children, onClose, errorMessage} = {...defaults, ...props}
```

Comment:
also you can use defaultProps and not this defaults construct https://facebook.github.io/react/docs/reusable-components.html#es6-classes